.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** if this is a _question_ about how to build / test / query / deploy using
+        Bazel intellij, or a _discussion starter_, send it to bazel-discuss@googlegroups.com or discuss on
+        [Bazel Slack](https://slack.bazel.build/) #intellij channel instead!
+  - type: textarea
+    id: desc
+    attributes:
+      label: >
+        Description of the bug:
+  - type: textarea
+    id: repro
+    attributes:
+      label: >
+        What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if
+        possible.
+  - type: input
+    id: os
+    attributes:
+      label: >
+        Which Intellij IDE are you using? Please provide the specific version.
+  - type: input
+    id: version
+    attributes:
+      label: >
+        What programming languages and tools are you using? Please provide specific versions.
+  - type: input
+    id: plugin
+    attributes:
+      label: >
+        What Bazel plugin version are you using?
+  - type: textarea
+    id: relevant-info
+    attributes:
+      label: Have you found anything relevant by searching the web?
+      description: >
+        Places to look:
+
+         * GitHub issues: https://github.com/bazelbuild/intellij/issues
+         * Email threads on https://groups.google.com/forum/#!forum/bazel-discuss
```

Comment:
I think we should also update this to https://groups.google.com/forum/#!forum/intellij-bazel-plugin